lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -27,13 +27,14 @@ def add_options_defaults(options)
             options[:type] ||= OvirtSDK4::DiskType::DATA
             options[:format] ||= OvirtSDK4::DiskFormat::COW
 
-            options[:sparse] = true unless options[:sparse].present?
+            # options[:sparse] = true unless options[:sparse].present?
```

Comment:
Please don't leave commented out code. If it's not needed anymore, you can delete it.

Reply:
This changes the behaviour of line 37:
```ruby
 options[:disk][:sparse] ||= options.delete(:sparse)
```

Now if the option is not set, it will pass `nil` to ovirt, as opposed to `true` that was set by the commented out line.